samples/java/hibernate/src/main/java/com/google/cloud/postgres/models/Tracks.java

```diff
@@ -0,0 +1,85 @@
+package com.google.cloud.postgres.models;
+
+import com.google.cloud.postgres.CurrentLocalDateTimeGenerator;
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import org.hibernate.annotations.GenerationTime;
+import org.hibernate.annotations.GeneratorType;
+
+@Entity
+public class Tracks {
+
+  @EmbeddedId private TracksId id;
```

Comment:
I think it would be good to add a comment here that explains what we are doing to model an interleaved table.